helevents/tests/test_gdpr_api_get.py

```diff
@@ -91,6 +103,16 @@ def _get_signup_group_profile_data(signup_group: Optional[SignUpGroup]) -> dict:
     return profile_data
 
 
+def _get_field_translations(obj: object, field_name: str) -> list:
+    return [
+        {
+            "key": f"{field_name}_{lang}".upper(),
+            "value": getattr(obj, f"{field_name}_{lang}"),
```

Comment:
Maybe we should skip translation which don't have a value for a specific language? Just thinking that like this well probably end up most translations being `None`.

Reply:
yes, your comment applies to the actual logic also. Yup perhaps the fields not having translation could just be left out 🤔